includes/data-explorer-container-system-properties.md

```diff
@@ -2,36 +2,49 @@
 author: orspod
 ms.service: data-explorer
 ms.topic: include
-ms.date: 01/08/2020
+ms.date: 02/27/2020
 ms.author: orspodek
 ---
 
 ### Event system properties mapping
 
-If you selected **Event system properties** in the **Data Source** section of the table above, go to the [Web UI](https://dataexplorer.azure.com/) to run the relevant KQL command for proper mapping creation.
+> [!Note]
+> * System properties are supported for single-record events.
+> * For `csv` mapping, properties are added at the beginning of the record. For `json` mapping, properties are added to according to name that appears in drop-down list.
 
-   **For csv mapping:**
+If you selected **Event system properties** in the **Data Source** section of the table above you need to include these properties in table schema and mapping.
 
-    ```kusto
-    .create table MyTable ingestion csv mapping "CsvMapping1"
+**Table schema example**
+If your data include 3 columns: `Timespan`, `Metric` and `Value`, And the properties you would like to include are `x-opt-enqueued-time` and `x-opt-offset`. Create or alter table schema as follows:
```

Comment:
includes
And the properties you include are `x-opt-enqueued-time` and `x-opt-offset`, create or alter table schema as follows: